packages/isar/lib/src/collection_schema.dart

```diff
@@ -3,7 +3,7 @@ part of isar;
 /// @nodoc
 @protected
 class CollectionSchema<OBJ> {
-  static const generatorVersion = 0;
+  static const generatorVersion = 1;
```

Comment:
May I know why we need to increment the generator from 0=> to 1

Reply:
This only check whether the user has old files that need to be regenerated :)